README.md

```diff
@@ -0,0 +1,20 @@
+# regexgame
+
+This game aims to improve a users understanding of regexes.
+
+## Build and run
+```bash
+docker compose up --build
+```
+then access
+[`http://localhost/minigames/regexgame`](http://localhost/minigames/regexgame/)
+
+## Run for development with hot reload
```

Comment:
Maybe we should mention VSCode as explicit requirement cause it is used for the development web server and the repo settings are pushed.